classes/class-wp-nr-apm.php

```diff
@@ -85,7 +85,7 @@ public function set_custom_variables() {
 		if ( function_exists( 'newrelic_set_user_attributes' ) ) {
 			if ( is_user_logged_in() ) {
 				$user = wp_get_current_user();
-				newrelic_set_user_attributes( $user->ID, '', array_shift( $user->roles ) );
+				newrelic_set_user_attributes( $user->ID, '', $user->roles[0] );
```

Comment:
Can we verify that the index of [0] actually exists? Just in case the roles are hosed.

Reply:
Thanks @allan23 , I updated the PR :)